.github/deploy.sh

```diff
@@ -27,35 +23,44 @@ mkdir out/master/
 cp util/gh-pages/index.html out/master
 python ./util/export.py out/master/lints.json
 
-if [ -n "$TRAVIS_TAG" ]; then
+if [[ -n "$TRAVIS_TAG" ]]; then
     echo "Save the doc for the current tag ($TRAVIS_TAG) and point current/ to it"
     cp -r out/master "out/$TRAVIS_TAG"
     rm -f out/current
     ln -s "$TRAVIS_TAG" out/current
 fi
 
 # Generate version index that is shown as root index page
-(
-    cp util/gh-pages/versions.html out/index.html
-
-    cd out
-    python -c '\
-        import os, json;\
-        print json.dumps([\
-            dir for dir in os.listdir(".")\
-            if not dir.startswith(".") and os.path.isdir(dir)\
-        ])' > versions.json
-)
+cp util/gh-pages/versions.html out/index.html
+pushd out
+
+cat <<-EOF | python - > versions.json
```

Comment:
IMO the `python -c` version is way easier to understand. With this I only can guess what it does. Does this change fix something?

I like the `pushd`/`popd` change though.

Reply:
Look at it again. I think we could > out/version.json directly, no need for cd and pushd.

Reply:
Also I like this version better because you could write python script in bash as in a python file.

Reply:
Ok, let's do the `<<-EOF` version with `out/version.json`.